docs/guides/integration-mobile-client-server.md

```diff
@@ -307,4 +307,48 @@ UID2Manager.shared.automaticRefreshEnabled = false
 
 ## Optional: UID2 Prebid Mobile SDK Integration
 
-<PrebidMobileSDK />
\ No newline at end of file
+<PrebidMobileSDK />
+
+## Error Response States
+
+[**GWH_IB_01 Are these states equally applicable to client-side and server-side?**]
+
+[**GWH_IB_02 Are these states equally applicable to Android and iOS, and identical? (if yes on both of these questions I will make this into a snippet when we've ironed out the wording)**]
+
+In certain conditions, the mobile SDK might return one of the following error response states:
+
+- [Expired](#response-state-of-expired)
+- [RefreshExpired](#response-state-of-refreshexpired)
+- [Invalid](#response-state-of-expired)
+- [NoIdentity](#response-state-of-expired)
+
+### Response State of Expired
+
+A response state of `Expired` means that the UID2 token has expired but the refresh token has not expired, therefore the UID2 token can be refreshed.
+
+If you have automatic refreshing enabled on the SDK, which is the default setting, the SDK should refresh the token for you. However, there are some scenarios where you might observe this state. For example:
+
+- If the app is launched and the SDK is immediately queried, you might see a response state of `Expired` while, in the background, the SDK is in the process of refreshing the token and will quickly update with the new identity. In this scenario, you should not need to generate a new token. [**GWH_IB_03 what SHOULD they do? Just wait, and they will see a success error response? Or...?**]
```

Comment:
>**GWH_IB_03 what SHOULD they do? Just wait, and they will see a success error response? Or...?**

If they have `automaticRefreshEnabled`, then they don't need to do anything. The SDK will **immediately** kick off a refresh as soon as the SDK is initialised. If they ignore and instead call `refreshIdentity`, they will likely end up refreshing the identity twice in quick succession.